src/components/ContactSalesForm/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { ReactComponent as CloseSvg } from '../../img/close.svg'
+import { FormState } from '../../types/FormState'
+import { sendEvent } from '../../utils/gtm'
+import classNames from 'classnames'
+import styles from './ContactSalesForm.module.scss'
+
+interface ContactSalesFormProps {
+  className?: string | string[]
+}
+export default function ContactSalesForm({ className }: ContactSalesFormProps) {
+  const submitEndpoint = process.env.FPJS_LEAD_URL
+
+  const [email, setEmail] = useState('')
+  const [website, setWebsite] = useState('')
+  const [formState, setFormState] = useState(FormState.default)
+  const [errorMessage, setErrorMessage] = useState('')
+
+  async function handleSubmit(e) {
+    e.preventDefault()
+
+    setFormState(FormState.loading)
+
+    const { ok, error } = await fetch(submitEndpoint, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        email,
+        website,
+        name: email,
+      }),
+    }).then((response) => response.json())
+
+    if (!ok) {
+      setErrorMessage(error.message || 'Something went wrong. Please try again later.')
+      setFormState(FormState.failed)
+      setTimeout(() => {
+        setFormState(FormState.default)
+      }, 2500)
+      sendEvent({ event: 'leadSubmit.error' })
+    } else {
+      setFormState(FormState.success)
+      sendEvent({ event: 'leadSubmit.success' })
+    }
+  }
+
+  return (
+    <form
+      className={classNames(
+        className,
+        styles.contactSalesForm,
+        { [styles.success]: formState === FormState.success },
+        { [styles.failed]: formState === FormState.failed },
+        { [styles.loading]: formState === FormState.loading }
+      )}
+      onSubmit={handleSubmit}
+    >
+      {(formState === FormState.default || formState === FormState.loading) && (
+        <div className={styles.form}>
+          <label htmlFor='email' className={styles.label}>
+            Email:
+            <input
+              type='email'
+              name='email'
+              required
+              className={styles.input}
+              placeholder='Enter your email'
+              value={email}
+              onChange={(e) => setEmail(e.target.value)}
+              disabled={formState === FormState.loading}
+            />
+          </label>
+          <label htmlFor='website' className={styles.label}>
+            Website:
+            <input
+              type='url'
+              name='website'
+              required
+              className={styles.input}
+              placeholder='Enter your website'
+              value={website}
+              onChange={(e) => setWebsite(e.target.value)}
+              disabled={formState === FormState.loading}
+            />
+          </label>
+
+          <Button
+            className={styles.button}
+            type='submit'
+            mobileIcon={<ChevronRightSvg className={styles.mobileIcon} />}
```

Comment:
We don't need it to become an Icon on a mobile version it looks like there is enough space for text even on iPhone 5

Reply:
That was just the common button component, I'll look into writing a custom class.